app/controllers/api/v2/compact_index_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V2::CompactIndexController < Api::BaseController
```

Comment:
how do you want those endpoints? ` /version` and `/info` directly? With class `::CompactIndexController`?